biocrnpyler/dna_part.py

```diff
@@ -56,13 +59,15 @@ def __init__(self,name,mechanisms=None,parameters=None,**keywords):
     @property
     def dna_species(self):
         return Species(self.name, material_type="dna")
+
     def __repr__(self):
         myname = self.name
         if(self.position is not None):
             myname+="_"+str(self.position)
         if(self.direction =="reverse"):
             myname += "_r"
         return myname
+    '''
```

Comment:
please remove unused code

Reply:
repr is useful for debugging and used in some example notebooks I think?